lib/Runtime/Language/JavascriptOperators.inl

```diff
@@ -31,6 +31,18 @@ namespace Js
         }
     }
 
+    __forceinline TypeId JavascriptOperators::GetTypeId(Js::RecyclableObject* aValue)
+    {
+        AssertMsg(aValue != nullptr, "GetTypeId aValue is null");
+
+        auto typeId = RecyclableObject::UnsafeFromVar(aValue)->GetTypeId();
```

Comment:
Is the UnsafeFromVar needed here?  Or is it there just to get the Assert in UnsafeFromVar?